tests/ui-toml/allow_mixed_uninlined_format_args/uninlined_format_args.rs

```diff
@@ -0,0 +1,182 @@
+// aux-build:proc_macro_with_span.rs
+// run-rustfix
```

Comment:
This file seems to be the same as the normal ui test one, just with the new config value. Usually, it's enough in the ui-toml tests, to just include the relevant test cases. So have a few that should be linted, and a few which still shouldn't be.

That also clearly indicates what this file actually tests. Like this I had to use a local diff to see that the test covered three examples from this file. Would you mind reducing the scope? Then you can also remove the `with_span` macro magic and so on from this test as it's covered by the other one.

Reply:
fixed